src/components/StatAll.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  Link,
+} from 'react-router-dom';
+
+const Stats = () => {
+  const stats = useSelector((state) => state.stats.stats);
+  const [filter, setFilter] = useState('All');
+
+  const tab = [{ id: 0, name: 'All' }];
+  stats.map((stat) => (
+    tab.push({
+      id: stat.id,
+      name: stat.calendarYear,
+    })
+  ));
+
+  function shortRevenue(revenue) {
+    return (parseInt(revenue, 10) / 1000000000).toFixed(2);
+  }
+
+  return (
+    <>
+      <ul className="filters">
+        {tab.map((option) => (
+          <button
+            type="button"
+            onClick={() => { setFilter(option.name); }}
+            key={option.id}
+            style={{ backgroundColor: filter === option.name ? '#ce4277' : 'white', color: filter === option.name ? 'white' : 'black' }}
+          >
+            { option.name }
+          </button>
+        ))}
+      </ul>
+      <section className="cards">
+
+        {stats.length > 0 && filter === 'All' && stats.map((stat) => (
+          <Link to={`/details/${stat.calendarYear}`} key={stat.id} className="card">
+            <div>
+              <p>{stat.calendarYear}</p>
+              <h3>{stat.symbol}</h3>
+              <h2>
+                $
+                { shortRevenue(stat.revenue)}
+                {' '}
+                bills
+              </h2>
+              <p>{stat.date}</p>
+            </div>
+          </Link>
+        ))}
+        {stats.length > 0 && filter !== 'All' && stats.filter((stat) => stat.calendarYear === filter).map((stat) => (
+          <Link to={`/details/${stat.calendarYear}`} key={stat.id} className="card">
+            <div>
+              <p>{stat.calendarYear}</p>
+              <h3>{stat.symbol}</h3>
+              <h2>
+                $
+                { shortRevenue(stat.revenue)}
+                {' '}
+                bills
+              </h2>
+            </div>
+          </Link>
```

Comment:
- [ ] As per the given design, the full details section also has two columns and the details are to the right. See the screenshots below and fix this. Your design has just one column and the details to the left.
Your design
![Screen Shot 2023-09-09 at 7 15 42 AM](https://github.com/MenendezON/react-capstone/assets/62849872/98527e80-9a08-4b47-9252-52210d5253d3)

Given design
![Screen Shot 2023-09-09 at 7 16 25 AM](https://github.com/MenendezON/react-capstone/assets/62849872/a7e10f82-7968-4665-8e38-e0d2b8beacb7)